lib/src/widgets/additional_signup_card.dart

```diff
@@ -0,0 +1,232 @@
+part of auth_card;
+
+class _AdditionalSignUpCard extends StatefulWidget {
+  _AdditionalSignUpCard({
+    Key? key,
+    required this.formFields,
+    required this.loginAfterSignUp,
+    required this.switchToLogin,
+    this.onSubmitCompleted,
+    this.loadingController,
+  }) : super(key: key) {
+    if (formFields.isEmpty) {
+      throw RangeError('The formFields array must not be empty');
+    } else if (formFields.length > 6) {
+      throw RangeError(
+          'More than 6 formFields are not displayable, you provided $formFields.length');
```

Comment:
We should mention that there is a max of 6 additional fields in the docs.

Reply:
done in 3e6a221